services/auth_service.py

```
@@ -9,36 +9,42 @@
+
+def verify_password(plain_password : str, hashed_password :str):
+    return pwd_context.verify(plain_password[:72], hashed_password)
 
 def signup_user(data: UserSignUp):
     users = get_users()
 
     if any(u["email"] == data.email for u in users):
         raise HTTPException(400, "이미 존재하는 이메일")
 
+    hashed_password = get_password_hash(data.password)
+    user_creat_at = datetime.now()
```

Ruff (0.14.13)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)